scripts/CSSRE/kafka-debug/script.sh

```diff
@@ -0,0 +1,213 @@
+#!/bin/bash
+
+set -o pipefail
+set -o nounset
+
+function main(){
+
+  local deploymentCount
+  local statefulsetCount
+  local kafkaBrokerRequired
+  local kafkaBrokerCount
+  local zookeeperBrokerRequired
+  local zookeeperBrokerCount
+  local adminServerRequired
+  local adminServerCount
+  local canaryRequired
+  local canaryCount
+  local kafkaExporterRequired
+  local kafkaExporterCount
+  local podCount
+  local podRequired
+  local podFailCount
+  local serviceCount
+  local routeCount
+  local pvCount
+  local pvcCount
+  local secretCount
+  local configmapCount
+  local totalCount
+
+  podFailCount=0
+  
+  echo "_______________________________________________________________"
+  echo " Kafka Resource Status" 
+  echo "_______________________________________________________________"
+
+  echo "_______________________________________________________________"
+  echo " Deployment/Statefulset Status" 
+  echo "_______________________________________________________________"
+ 
+  deploymentCount=$(oc get deployments -n "${resource_namespace}" | tail -n +2 | wc -l | sed 's/^ *//g')
```

Comment:
I do see that kafka namespaces have labels attached like `app.kubernetes.io/managed-by: kas-fleetshard-operator`
Can we do a check that the passed namespace is in the list of `oc get ns -l app.kubernetes.io/managed-by=kas-fleetshard-operator`
If that is the case that it will always have that label